src/iconc/cmain.c

```diff
@@ -124,7 +124,16 @@ char **argv;
       refpath = patchpath+18;
 #endif					/* ExpTools */
    else
-     refpath = relfile(argv[0], "/../");
+      refpath = relfile(argv[0], "/../");
+
+     if (refpath[strlen(refpath)] != '/') {
```

Comment:
If this is commented-out code that is going away, why is it listed in green with + signs to the left of it and no indication of it being commented?  Perhaps it is a user interface issue.